doc/source/release_1_0.rst

```diff
@@ -0,0 +1,77 @@
+.. _release_1_0:
+
+Breaking changes in release 1.0
+===============================
+
+This document outlines the breaking changes expected in the upcoming major version `1.0` of PyAEDT.
```

Comment:
```suggestion
This page describes the breaking changes expected in the upcoming major version 1.0 of PyAEDT.
```
The single backtick that was around `1.0` would be rendered as italic in the doc. I don't think this is necessary.